### ISSUE TITLE
Debian and Ubuntu user older repositories

### ISSUE THREAD

**Issue by freebrowser1** [CypherpunkArmory/UserLAnd#825]
# Describe the bug

When installing a fresh Debian or Ubuntu, it appears older and outdated versions of installed apps are used.

# Steps to reproduce the behavior:

1. Install a clean Userland and select Ubuntu.
2 Open a terminal (can be an SSH client on another computer in the same wifi network for typing convenience), log in and run :

```
sudo apt install nano wget youtube-dl imagemagick
sudo apt update && sudo apt upgrade
```
Both steps together take a long time (up till 30 minutes), but that is not the issue as on a desktop computer or server it is barely longer.

3. Run youtube-dl --version
This issues 2018.03.14 which is very outdated.
And 'sudo apt install youtube-dl' again, issues 'is already the latest version'.

Run convert -version
This issues version 6.8.xx while the current version is 7.0.8. Same with re-installing as with youtube-dl.

It appears to be Ubuntu Bionic 18.04.1 as seen during package install, but the packages are very outdated. The command uname -a issues archlinux and not Ubuntu or Debian which is also weird.

When installing Debian and installing these packages, the same old version are installed and issues occur.
# Screenshots
N/A

# Device Information

Device: Huawei Mate 9.  
Android Version: Pie 9.0.  
UserLAnd Version 2.5.8
Not rooted.
Runs stock ROM.

**Comment by Herzogs** [comment]
I do not think that UserLAnd problem since they use the isos that these distros provide and the one responsible for updating the packages are from the distros.

**Comment by lewisdonofrio** [comment]
keep in mind I've read tickets that have issues and they are "Huawei" devices. "mileage may vary"

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
